spec/factories/wikies.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :wiky do
+    title "MyString"
+body "MyText"
+user nil
+  end
+
+end
```

Comment:
In factories, and in tests, try to use realistic test data. `"MyString"` and `"MyText"` aren't realistic values for a wiki title and body. Realistic data can help aid debugging and sometimes help uncover previously hidden bugs. "Favorite Ruby Gems" and "Devise, pry, and shoulda-matchers are some gems I like." would be more realistic values in this example.